src/routes/root.ts

```diff
@@ -0,0 +1,11 @@
+import express, { Router } from "express";
+
+export default function createRootRouter(): Router {
```

Comment:
This doesn't need to be wrapped in a function. Other routes are wrapped because we need access to the node-postgres `client` object